app/mac/BUILD.gn

```diff
@@ -54,7 +54,7 @@ mac_app_bundle("electron_app") {
   ]
 
   sources = [
-    "//electron/chromium_src/chrome/app/chrome_exe_main_mac.c",
+    "//electron/chromium_src/chrome/app/chrome_exe_main_mac.cc",
```

Comment:
doesn't this also need the seatbelt dep?

Reply:
I referenced upstream `chrome/BUILD.gn` and it doesn't seem to be needed
```
  mac_app_bundle("chrome_app") {
    output_name = chrome_product_full_name

    info_plist_target = ":chrome_app_plist"
    extra_substitutions = [
      "CHROMIUM_BUNDLE_ID=$chrome_mac_bundle_id",
      "CHROMIUM_SHORT_NAME=$chrome_product_short_name",
      "CHROMIUM_CREATOR=$chrome_mac_creator_code",
    ]

    sources = [
      "app/chrome_exe_main_mac.cc",
    ]

    extra_configs = [ "//build/config/compiler:wexit_time_destructors" ]

    deps = [
      ":chrome_app_strings_bundle_data",
      ":chrome_resources",
      ":chrome_versioned_bundle_data",
      "//chrome/common:version_header",
    ]
...
```

Reply:
mainly it only uses seatbelt in helper's code block
```
if defined(HELPER_EXECUTABLE)
```